frontend_tests/node_tests/stream_data.js

```diff
@@ -635,3 +641,18 @@ zrequire('marked', 'third/marked/lib/marked');
 
     assert.equal(page_params.notifications_stream, "foo");
 }());
+
+(function test_invite_streams() {
+    // add default stream
+    var orie = {
+        stream_id: 320,
+        name: 'Orie',
+        subscribed: true,
+    };
+
+    stream_data.add_sub('Orie', orie);
+    stream_data.set_realm_default_streams([orie]);
+
+    var expected_list = ['India', 'Antarctica', 'Orie'];
```

Comment:
I'm not sure where "India" and "Antarctica" (this is spelled wrong) come from. I think it's from further up in the code, but like what Steve said, you might want to re-initialize the data and do a specific setup.

Reply:
Any way to clear previous stuff?
BTW This came from `stream_data.subscribe_streams()`

Reply:
`stream_data.clear_subscriptions();`

Reply:
Updated.